CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+Contributing
+-----
+
+### Requirements
+
+```r
+install.packages(c('httptest', 'testthat', 'mockthat', 'getPass', 'rstudioapi', 'mockery', 'lintr', 'styler', 'devtools', 'roxygen2'))
+```
+
+### Development
+
+This project follows the [tidyverse style
+guide](https://style.tidyverse.org/documentation.html) and uses
+[roxygen2](https://github.com/r-lib/roxygen2) for generating documentation.
+
+External functions need an `@export` tag in the Roxygen2 docstring. They must
+also be added to `_pkgdown.yml`.
+
+After adding a new function, you must:
+```r
+library('roxygen2')
+roxygenise()
+```
```

Comment:
(IDK if you're using RSTudio, but this is also available as ctrl-shift-D or Build -> Document.)